### PR TITLE
docs: add contribute section

### PR DESCRIPTION
**Pull request by unional**
also clean up readme to conform with markdown lint
and remove `dep`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
-# Co-Coders API 
+# Co-Coders API
+
 [![Build Status](https://travis-ci.org/clean-code-projects/co-coders-api.svg?branch=master)](https://travis-ci.org/clean-code-projects/co-coders-api)
 
+## contribute
+
```
Comment (akessner) [inline README.md:6]: Let's add a line about needing to replicate the folder structure with github.

**Review by akessner — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (unional) [reply README.md:6]: I don't know what's the difference but I actually don't need to follow the folder structure.
```sh
laptop:co-coders-api hwong$ go test ./...
?   	github.com/clean-code-projects/co-coders-api/internal/api	[no test files]
?   	github.com/clean-code-projects/co-coders-api/internal/co_coders/criteria[no test files]
ok  	github.com/clean-code-projects/co-coders-api/internal/co_coders/match	(cached)
?   	github.com/clean-code-projects/co-coders-api/internal/co_coders/user	[no test files]
laptop:co-coders-api hwong$ pwd
/Users/hwong/code/clean-code-projects/co-coders-api
```

Ran `mv clean-code-projects ccp` and it still works.

Comment (akessner) [reply README.md:6]: Nice, go modules fixes those issues.

**Review by akessner — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+## contribute
+
+```sh
+# test
+go test -v ./...
+```
+
 ## development environment
 
 We will be primary using VS Code.
```
Comment (pajatopmr) [inline README.md]: "primarily be" rather than "primary".

**Review by pajatopmr — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (akessner) [reply README.md]: You can add a suggestion. like this:
```suggestion
We will primarily be using VS Code.

```

**Review by xstrengthofonex — APPROVED** [review]

**Pull request merged**